diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -870,6 +871,7 @@ export class ContentGame extends React.Component {
                 recipient: recipient,
                 message: body,
                 truth: deception,
+                message_type: message_type,
```

Comment:
The field isn't being populated properly because it's just called `type`:

```suggestion
                type: message_type,
```